src/Pangraph.hs

```diff
@@ -97,27 +97,25 @@ verifyGraph vs = mapMaybe (\e -> lookupEndpoints (e, edgeEndpoints e))
 makeEdge :: [Attribute] -> (Vertex, Vertex) -> Edge
 makeEdge = Edge Nothing
 
--- | Vetex constuctor
+-- | Vetex constructor
 makeVertex :: VertexID -> [Attribute] -> Vertex
 makeVertex = Vertex
 
 -- * Pangraph Getters
 
--- | Returns the Edges from a Pangraph instance
+-- | Returns the ['Edge'] from a Pangraph instance
 edges :: Pangraph -> [Edge]
 edges p = Map.elems $ edges' p
 
---  | Returns the vertices from a Pangraph instance
+-- | Returns the ['Vertex'] from a Pangraph instance
 vertices :: Pangraph -> [Vertex]
 vertices p = Map.elems $ vertices' p
 
--- | Lookup of the EdgeID in a Pangraph.
--- | Complexity: /O(log n)/
+-- | Lookup of the EdgeID in a Pangraph. Complexity: /O(log n)/
 edgeByID :: EdgeID -> Pangraph -> Maybe Edge
```

Comment:
What about naming this `lookupEdge` instead? I think this is a more typical name for such function.